doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc

```diff
@@ -65,7 +69,7 @@ In some authoring environments, you might need to ensure that several users can
 +
 [NOTE]
 ====
-Do not enable the controller-based startup strategy in an environment with a high-availability {CENTRAL}. 
+Do not enable the controller-based startup strategy in an environment with a high-availability {CENTRAL}.
```

Comment:
If startupstrategy is code, then please use code formatting Othewise I don't see why you removed the space.

Reply:
The removed space is after the full stop. The one in "startup strategy" remains here.